src/components/PageTitles.js

```diff
@@ -41,15 +41,32 @@ const PageTitles = {
   defaultPageTitle: "Register a Food Business"
 };
 
-PageTitles.getUrlPageTitle = (url) => {
+PageTitles.getUrlPageTitle = (url, validatorErrors, allValidationErrors) => {
+  var allValidationErrorLength =
```

Comment:
If we pass in validatorErrorsCleaned and allValidationErrorsCleaned as suggested in comment on SessionWrapper, then we know these are definitely objects and so should be able to simplify lines 45-56 to just something like:
`var isError = Object.keys(allValidationErrors).length > 1 || Object.keys(validatorErrors).length > 1`